core/src/main/java/com/linecorp/armeria/common/util/Releasable.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2017 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.common.util;
+
+/**
+ * A holder of releasable resource {@link T}.
+ */
+public interface Releasable<T> {
```

Comment:
`ReleasableHolder`? I wouldn't expect a `Releasable` to have a `get` method (or be anything more than an `Closeable` which sounds very similar).